docs/site/Authentication-overview.md

```diff
@@ -0,0 +1,68 @@
+---
+lang: en
+title: 'LoopBack 4 Authentication Overview'
+keywords: LoopBack 4.0, LoopBack 4, Authentication
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Authentication-overview.html
+---
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces:
+
+- Authentication
+- Authorization
+
+Authentication is a process of verifying someone's identity before a protected
+resource is accessed.
+
+Authorization is a process of deciding if a user can perform an action on a
+protected resource.
+
+{% include note.html content=" For a description of an Authorization process, please see [Authorization](Loopback-component-authorization.md). " %}
+
+This document gives you an overview of the authentication system provided in
+LoopBack 4.
+
+Let's start with the following scenario: Suppose you want to secure endpoint
+`GET /todo` using
+[JWT(JSON web token)](https://github.com/auth0/node-jsonwebtoken#readme)
+strategy, the diagram below shows how such authentication process works with
+LoopBack's authentication mechanism.
+
+![authentication_overview_request_handle_flow](./imgs/authentication/authentication-overview.png)
```

Comment:
there're extra icons at the bottom right. Could you please remove that?

Reply:
The diagram is created kinda in a rush, I will draw it again.

Reply:
@dhmlau I updated the diagram.